cloud_watch_client.py

```diff
@@ -153,13 +153,19 @@ def get_alarms(self, state="ALARM"):
         res = []
         my_conf = dict(
             StateValue=state,
+            MaxRecords=100
         )
         if self._namespace:
             # Cloud watch does not accept an empty sting for AlarmNamePrefix
             my_conf['AlarmNamePrefix'] = self._get_full_normalized_alarm_name('')
         try:
-            resp = self._client.describe_alarms(**my_conf)
-            res = resp.get('MetricAlarms')
+            resp = None
+            next_token = None
+            while resp is None or next_token is not None:
```

Comment:
if resp is None, line 166 will catch an error. we might not need to add resp in a while condition.
instate of that, we might convert it like this.
`next_token = True
while next_token is not None:`
if we might have a while true loop problem, we might add to the logger to catch it.